.dockerignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 client/
 workspace/
+
+.balena/secrets/
```

Comment:
@vipulgupta2048 You need to place `.balena/secrets/.npmrc` file with a string like
```
//registry.npmjs.org/:_authToken=PERSONAL_TOKEN
```